packages/preview/ruc-thesis/0.1.0/README.md

```diff
@@ -0,0 +1,144 @@
+# 中国人民大学本科生毕业论文 Typst 模板
+
+![License](https://img.shields.io/badge/license-MIT-blue.svg)
+![Typst](https://img.shields.io/badge/built%20with-Typst-239dad.svg)
+
+这是一个非官方的中国人民大学（RUC）本科生毕业论文 Typst 模板。旨在帮助同学们用现代化的工具，高效、优雅地完成论文排版。

```

Comment:
Same for this introduction sentence, it would be nice to provide an English translation. The rest of the README can keep using Chinese.